File: wireless/sims/instance-post-example-1/instance-post-example-1.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Twilio;
using Twilio.Rest.Wireless.V1;
using Twilio.Http;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account SID and Auth Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var sim = SimResource.Update("DEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
            callbackMethod: HttpMethod.Post,
            commandsCallbackUrl: new Uri("https://sim-manager.mycompany.com/sim-update-callback/AliceSmithSmartMeter"),
            status: SimResource.StatusEnum.Active);

        Console.WriteLine(sim.Sid);
    }
}

File: wireless/sims/instance-post-example-1/instance-post-example-1.5.x.js
// Download the Node helper library from twilio.com/docs/node/install
// These consts are your accountSid and authToken from https://www.twilio.com/console
// To set up environmental variables, see http://twil.io/secure
const accountSid = process.env.TWILIO_ACCOUNT_SID;
const authToken = process.env.TWILIO_AUTH_TOKEN;
const client = require('twilio')(accountSid, authToken);

client.wireless
  .sims('DEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA')
  .update({
    callbackMethod: 'POST',
    callbackUrl:
      'https://sim-manager.mycompany.com/sim-update-callback/AliceSmithSmartMeter',
    status: 'active',
  })
  .then(response => {
    console.log(response);
  });
